openjdk8/src/main/docker/Dockerfile

```diff
@@ -25,18 +25,35 @@ RUN \
  && apt-get -y -q --no-install-recommends install \
     # install the jdk and its dependencies
     ca-certificates-java \
-    openjdk-${openjdk.version.major}-jdk-headless=${openjdk.version}'*' \
     # procps is used in the jvm shutdown hook
     procps \
     # other system utilities
     netbase \
     wget \
     unzip \
+    fontconfig \
```

Comment:
Do you remember which command requires this? We want to keep this list of package as small as possible. If required, maybe add a comment?

Reply:
Would keep it, was there in the adoptium dockerfile.

Reply:
I added fontconfig and locales per Ludo's recommendation. Don't think it is required right now. @ludoch  - thoughts on removing this?